api/api.py

```diff
@@ -270,7 +269,11 @@ def chunked_completion(self, completion_request: CompletionRequest):
         )
         generator_args = GeneratorArgs(
             completion_request.messages[-1].get("content"),
+            max_new_tokens=(
+                completion_request.max_tokens if completion_request.max_tokens else 16
```

Comment:
Define DEFAULT_MAX_TOKENS at the top and set it to 16 then
max_new_tokens = completion_request.max_tokens or DEFAULT_MAX_TOKENS